libs/sdk-backend-spi/src/common/settings.ts

```diff
@@ -161,6 +161,11 @@ export interface ISettings {
      */
     enableCompanyLogoInEmbeddedUI?: boolean;
 
+    /**
+     * Indicates whether the insecure javascript events are enabled.
+     */
+    insecureJavascriptEventsEnabled?: boolean;
```

Comment:
i suggest not to publish this setting :)

Reply:
Removed